images/java/src/main/java/io/iron/lambda/Launcher.java

```diff
@@ -15,13 +15,29 @@
     public static void main(String[] args) {
         String handler = args[0];
         String payload = "";
-        try {
-            String file = System.getenv("PAYLOAD_FILE");
-            if (file != null) {
+        String file = System.getenv("PAYLOAD_FILE");
+        if (file != null) {
```

Comment:
What happens if `PAYLOAD_FILE` is set to empty? I mean, the env var is set, but the content is empty.